grails-app/controllers/org/broadinstitute/orsp/IndexController.groovy

```diff
@@ -5,7 +5,7 @@ class IndexController {
     UserService userService
 
     def index() {
-        render(view: "/index")
+        render(view: "/mainContainer/index")
```

Comment:
index() method, other methods() an IndexController itself seems to be not required anymore. Remove it

Reply:
will defer this until related PR's are merged